csharp/nuget/Facebook.Yoga.Native.nuspec

```diff
@@ -0,0 +1,38 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>Facebook.Yoga.Native</id>
+    <version>3.0.0-pre</version>
+    <title>Facebook.Yoga.Native</title>
+    <authors>Facebook</authors>
+    <owners>Facebook</owners>
+    <licenseUrl>https://github.com/facebook/css-layout/blob/master/LICENSE</licenseUrl>
+    <projectUrl>https://github.com/facebook/css-layout</projectUrl>    
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>A subset of CSS's flexbox layout algorithm and box model.</description>
+    <copyright>Copyright 2016 Facebook</copyright>
+    <tags>native flexbox flex-box css layout css-layout yoga facebook</tags>
+    <dependencies>
+      <group targetFramework=".NETStandard1.0" />
+    </dependencies>
+  </metadata>
+
+  <files>
+    <!-- build -->
+    <file src="Facebook.Yoga.Native.targets" target="build\netstandard1.0"/>
+
+    <!-- lib -->
+    <file src="_._" target="lib\netstandard1.0"/>
```

Comment:
The inclusion of _._ as the "lib" allows us to reference this nuget package in a .NET project... This is the model used by sqlite.native